### DISCUSSION TITLE
Utilization greater than 100%

### DISCUSSION THREAD

**Discussion by GSBhatta** [The-OpenROAD-Project/OpenROAD-flow-scripts#2030] [General]
> I was trying to get the fabric for a CLB. I am facing an issue with global_skip_io. It says that the utilization is more than 100%. As seen in the image below the utilization is 119.73%. I tried to change the Die_Area and Core_Area in the config.mk file. In fact i tried tomodify them in 2 different config files. they are
- ~/Desktop/OpenROAD-flow-scripts/flow/platforms/sky130hd/config.mk
- ~/Desktop/OpenROAD-flow-scripts/flow/designs/sky130hd/<topmodule>/config.mk
<img width="855" alt="image" src="https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/113631826/d9731498-66c3-4e95-b675-3ec4903d24ad">
Please guide me towards the right path so that I can get the output.

**Comment by maliberty** [comment]
Giving paths to local files doesn't show anything.  Please say what you changed and how the output changed?  It appears you have too small a core area.
